components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/HTTPGetFunctionImpl.java

```diff
@@ -39,10 +41,38 @@ public HTTPGetFunctionImpl() {
     }
 
     @Override
-    public void httpGet(String epUrl, Map<String, Object> eventHandlers) {
+    public void httpGet(String epUrl, Object... params) {
+
+        Map<String, Object> eventHandlers;
+        Map<String, String> headers = new HashMap<>();
+
+        switch (params.length) {
+            case 1:
+                if (params[0] instanceof Map) {
```

Comment:
Shall we add more debug logs to each switch case ?, Having a debug logs will give valuable insight about the logic and this will help to troubleshooting the issues as well.